### PR TITLE
Allow describing database views

### PR DESCRIPTION
**Pull request by pimeys**
A part of effort in https://github.com/prisma/migrations-team/issues/192

### CODE REVIEW

File: libs/sql-schema-describer/src/lib.rs [hunk 1]
```diff
@@ -446,6 +455,15 @@ pub struct Sequence {
...
 }
 
+/// An SQL view.
+#[derive(PartialEq, Debug, Clone)]
+pub struct View {
+    /// Name of the view.
+    pub name: String,
+    /// The SQL definition of the view.
+    pub definition: String,
+}
```
Comment (tomhoule) [inline libs/sql-schema-describer/src/lib.rs:465]: :+1:

File: libs/sql-schema-describer/tests/mssql_describer_tests.rs [hunk 2]
```diff
@@ -5,10 +5,50 @@ use crate::mssql::*;
...
+            SELECT b_id
+            FROM {0}.b"#,
+        db_name
+    );
+
+    conn.raw_cmd(&create_view).await.unwrap();
+
+    let inspector = SqlSchemaDescriber::new(conn);
+    let result = inspector.describe(db_name).await.expect("describing");
+    let view = dbg!(result).get_view("ab").expect("couldn't get ab view").to_owned();
```
Comment (tomhoule) [inline libs/sql-schema-describer/tests/mssql_describer_tests.rs:46]: stray dbg

File: libs/test-setup/src/connectors/mssql.rs [hunk 3]
```diff
@@ -39,6 +56,7 @@ pub async fn reset_schema(conn: &dyn Queryable, schema_name: &str) -> Result<(),
         schema_name
     );
 
+    conn.raw_cmd(&drop_views).await?;
     conn.raw_cmd(&drop_fks).await?;
     conn.raw_cmd(&drop_tables).await?;
```
Comment (tomhoule) [inline libs/test-setup/src/connectors/mssql.rs:61]: Random thought: would concatenating these queries and sending them all at the same time work?

**Review by tomhoule — APPROVED** [review]
amazing!

File: libs/test-setup/src/connectors/mssql.rs (hunk 3 above)
Comment (pimeys) [reply libs/test-setup/src/connectors/mssql.rs:61]: No. It will not work. I tried, but the SQL Server told me to try again.

**Pull request merged**